sh/install.sh

```diff
@@ -92,33 +110,31 @@ function install_python() {
 }
 
 function remove_python() {
-
+    PIP_PATH=$1
     # wavefront cli
-    pip uninstall wavefront-cli -y
+    $PIP_PATH uninstall wavefront-cli -y
     # pip
-    pip uninstall pip -y
+    $PIP_PATH uninstall pip -y
 
     # python
     if [ $OPERATING_SYSTEM == "DEBIAN" ]; then
+        remove_pkg python3-distutils
         echo "Uninstalling Python using apt-get"
-        apt-get remove python -y &> ${INSTALL_LOG}
+        apt-get remove python3 -y &> ${INSTALL_LOG}
         apt-get autoremove -y &> ${INSTALL_LOG}
     elif [ $OPERATING_SYSTEM == "REDHAT" ]; then
         echo "Uninstalling Python using yum"
-        yum remove python -y &> ${INSTALL_LOG}
+        yum remove python3 -y &> ${INSTALL_LOG}
     elif [ $OPERATING_SYSTEM == "openSUSE" ] || [ $OPERATING_SYSTEM == "SLE" ]; then
         echo "Uninstalling Python using zypper"
-        zypper remove python &> ${INSTALL_LOG}
+        zypper remove python3 &> ${INSTALL_LOG}
     fi
 
 }
 
 
 function install_pip() {
-    $PYTHON_PATH=$(which python) 2> /dev/null
-    if [ $OPERATING_SYSTEM == "openSUSE" ]; then
-       $PYTHON_PATH=$(which python2.7) 2>/dev/null
-    fi
+    PYTHON_PATH=$1
```

Comment:
Why don't we simply `apt install python3-pip` here in order to install proper system pip package?